src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -504,6 +517,15 @@ public BiomeGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		}
 	}
 
+	private void updateBiomeTreesForm() {

```

Comment:
Make these enabled only if custom trees is checked, not enabled true in all cases.